core/data/src/main/kotlin/com/yapp/fitrun/core/data/AuthRepositoryImpl.kt

```
@@ -0,0 +1,94 @@
+            tokenManager.saveLoginInfo(
+                accessToken = loginResponse.tokenResponse.accessToken,
+                refreshToken = loginResponse.tokenResponse.refreshToken,
+            )
+            Result.success(loginResponse.toDomainModel())
+        } catch (e: HttpException) {
+            // HTTP 에러 처리
+            val errorBody = e.response()?.errorBody()?.string()
+            val errorResponse = errorBody?.let {
+                try {
+                    json.decodeFromString<ErrorResponse>(it)
+                } catch (parseError: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)